README.md

```diff
@@ -2,12 +2,16 @@
 
 Sample App to display the weather using Yahoo weather API
 
+## Requirements
+- Rails application to show Yahoo weather information for 2 locations: Sydney and Melbourne.
+- The user should be able to select the location of their choice (Sydney or Melbourne).
+- After selecting the location, the user should be shown the forecast for the next few days.
+- Application should use models that will best represent the data being shown.
+- Each time the weather is checked, the results should be saved to the database. The user doesn’t need to be shown the results of previous weather checks, but they should be stored for future features which may be added to the application. 
+
 ## Assumptions
 - Only using Yahoo to get weather info, so only handling their style of data.
-- Not sure if it's useful to save the weather info, so I just saved it as json (rather than normalised tables).  
-
-## Regrets
-- I am used to rspec but went with minitest. Took a while to get used it :)
+- Not sure if it's useful to save the weather details, so I just saved it as json (rather than normalised tables).  
```

Comment:
I would tend towards saving in normalised tables because it means the data is processed (converted) once and enables queries to be performed.

Reply:
Yeah, I wish I did that, especially after I reread the requirements :)